back-end/etraceflask.py

```diff
@@ -130,8 +134,7 @@ def api_mac_post():
         return '["false","unknown"]'
 
     try:
-        conn = psycopg2.connect(database="sunhaobomac", user="postgres", password="postgres", host="127.0.0.1",
-                                port="5432")
+        conn = psycopg2.connect(database="sunhaobomac", user="postgre", password="postgre", host="127.0.0.1", port="5432")
```

Comment:
Actually "postgres" is the default username and password for PostgreSQL so maybe we should leave it as before since this is going public, name for the database will be fixed maybe later.